recommended-workflow.md

```diff
@@ -0,0 +1,13 @@
+# Recommended Proposal Workflow
+
+1. Write a document detailing the problem you are solving, and the requirements to solve the problem.
+    - Schedule a meeting with the relevant workgroup to discuss the idea
+2. Write a specification working with several other people from the workgroup
+3. Write example implementation
+4. Work to attract quality peer reviewers.  (The importance of this can not be understated, if no one is looking at a proposal, it is not going to be implemented anywhere).
+5. Develop a rigorous test plan.  
+    - It should test that it does what it says
+    - It should test that it doesn't do other things
+    - Test performance
+    - Test economics (designing these tests is complex and it is where the peer-review becomes especially needed.
```

Comment:
is testing economics even possible?(honest question) 

nit: missing closing bracket